opm/input/eclipse/Schedule/Action/State.hpp

```diff
@@ -20,74 +20,142 @@
 #ifndef ACTION_STATE_HPP
 #define ACTION_STATE_HPP
 
+#include <cstddef>
 #include <ctime>
 #include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <utility>
+#include <vector>
 
-#include <opm/input/eclipse/Schedule/Action/ActionResult.hpp>
+namespace Opm::RestartIO {
+    struct RstState;
+} // namespace Opm::RestartIO
 
-namespace Opm {
+namespace Opm::Action {
 
-namespace RestartIO {
-struct RstState;
-}
+    class ActionX;
+    class Actions;
+    class PyAction;
+    class Result;
 
-namespace Action {
+} // namespace Opm::Action
 
-class ActionX;
-class Actions;
-class PyAction;
+namespace Opm::Action {
 
-class State {
-
-struct RunState
+/// Management information about the current run's ACTION system, especially
+/// concerning the number of times each action has triggered/run and the
+/// last time it did run.
```

Comment:
was run?

Reply:
> was run?

Agreed.  I've pushed an update to that effect.